test/controllers/WhatIsYourRoleAsImporterControllerSpec.scala

```diff
@@ -63,6 +74,8 @@ class WhatIsYourRoleAsImporterControllerSpec extends SpecBase with MockitoSugar
           messages(application)
         ).toString
       }
+
+      verifyNoInteractions(mockAuditService)
```

Comment:
As this is an entirely different endpoint I think verifying there are no interactions here is a little redundant